src/os/bluestore/BlueStore.cc

```diff
@@ -12196,14 +12201,19 @@ void BlueStore::_kv_sync_thread()
       int r = cct->_conf->bluestore_debug_omit_kv_commit ? 0 : db->submit_transaction_sync(synct);
       ceph_assert(r == 0);
 
-#ifdef WITH_BLKIN
       for (auto txc : kv_committing) {
+#ifdef HAVE_JAEGER
+  if (txc->span)
+    jaeger_tracing::child_span("db sync submit", txc->span);
```

Comment:
you can also add metadata, that we had for blkin using logs `txn->span->log()`
like: https://github.com/ceph/ceph/blob/01997e76fed578cd667084bf5e2f0adace0c852a/src/osd/OSD.cc#L9648-L9654
for https://github.com/ceph/ceph/pull/41719/files#diff-6f3b1a5dcf5313dc06f71ba4997ee28ec219ecb5633a728b916465c684f1d87dR12213